modules/tinymce/src/plugins/lists/test/ts/browser/ListPropertiesTest.ts

```diff
@@ -38,7 +38,7 @@ describe('browser.tinymce.plugins.lists.ListPropertiesTest', () => {
 
   const updateDialog = (editor: Editor, currentValue: string, newValue: string) => {
     const doc = SugarDocument.getDocument();
-    FocusTools.isOnSelector('Check focus is on the input field', doc, inputSelector);
+    FocusTools.isOn('Check focus is on the input field', getInputSelector(doc));
```

Comment:
Likewise, this could've been `FocusTools.isOnByLabel`. My suggestion is to keep the agar API as easy to use as it was before, without needing to pass in elements (which often mandates `getOrDie`).